crypto-ffi/bindings/js/test/CoreCrypto.test.ts

```diff
@@ -380,13 +382,22 @@ describe("proteus", () => {
         );
         expect(decryptedMessage).toBe(message);
 
-        await expect(
-            newProteusSessionFromMessage(ALICE_ID, BOB_ID, SESSION_ID, message)
-        ).rejects.toThrowError(
-            // wdio wraps the error and prepends the original message with
-            // the error type as prefix
-            new Error("CryptoError: PreKeyStoreNotFound: 10")
-        );
+        let didfail = false;
+        try {
+            await newProteusSessionFromMessage(
+                ALICE_ID,
+                BOB_ID,
+                SESSION_ID,
+                message
+            );
+        } catch (e) {
+            didfail = true;
+            const error = e as Error;
+            expect(error.message).toBe(
+                "ProteusErrorOther: Another Proteus error occurred but the details are probably irrelevant to clients"
+            );
+        }
+        expect(didfail).toBeTruthy();
```

Comment:
The pattern before was more concise IMO, and should be equivalent.
So you could keep that pattern and just replace the expected error with
```
new Error("ProteusErrorOther: Another Proteus error occurred but the details are probably irrelevant to clients")
```

Reply:
Huh, you're right. At the time I thought I had a good reason to change it, but your suggestion works, so I'm going to use that.